vendor/github.com/kata-containers/agent/pkg/types/types.pb.go

```diff
@@ -11,6 +11,7 @@
 		IPAddress
 		Interface
 		Route
+		ARPNeighbor
```

Comment:
Missing commit to update `Gopkg.toml`?

Reply:
I have vendored in the agent changes. This is fixed now.